Note the requirement to grant access to orgs when authorizing the OAuth app (#46)
slack-samples/deno-github-functions: README.md

$ slack external-auth add
```

> To access repositories in an organization from a workflow, also grant access to
> the organization when authorizing your OAuth app.

Reviewer: ```suggestion
**Note: when working with repositories that are part of an organization, be sure to grant access to
that organization when authorizing your OAuth app.**
```

Author: This reads much better, thank you!
